.github/ISSUE_TEMPLATE/Bug-Report.yml

```diff
@@ -0,0 +1,68 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
```

Comment:
Since we're setting `placeholder` we can probably drop `value`.

Reply:
This is all mostly an adaptation of the form from https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms. I agree the placeholder and value seem redundant, I'm tempted to keep it for the moment just to see what it looks like. I wish there was a "preview" option somewhere.

Reply:
Actually, nm. Looking at screenshot in the example it definitely seems redundant so I'll get rid of `value`.